MaterialComponents.podspec

```diff
@@ -181,7 +181,7 @@ Pod::Spec.new do |mdc|
 
     component.test_spec 'tests' do |tests|
       tests.test_spec 'unit' do |unit_tests|
-        unit_tests.source_files = "components/#{component.base_name}/tests/unit/*.{h,m,swift}", "components/#{component.base_name}/tests/unit/supplemental/*.{h,m,swift}"
+        unit_tests.source_files = "components/#{component.base_name}/tests/unit/*.{h,m,swift}", "components/#{component.base_name}/tests/unit/supplemental/*.{h,m,swift}", "components/#{component.base_name}/examples/*.{h,m,swift}", "components/#{component.base_name}/examples/supplemental/*.{h,m,swift}", "components/#{component.base_name}/examples/tests/unit/*.{h,m,swift}"
```

Comment:
Should we create an issue to revert this change after this lands as including examples in the source files will no longer be needed?

Reply:
Hmm, I wonder if we should just add these to an independent spec or something.

@jverkoey who is a Cocoapods wiz and @andrewoverton who has been working on our test spec recently.

We have a Prototype/Alpha (proper Alpha) component in `BottomNavigation/examples` and want to add unit tests. What's the best way to do so?

Reply:
This is removing the examples from the MaterialComponentsExamples development pod in Xcode:

![screen shot 2018-12-18 at 10 01 52 am](https://user-images.githubusercontent.com/1753199/50162337-1ca95280-02ac-11e9-9451-8cc66c6aa35f.png)